requirements-dev.txt

```diff
@@ -13,9 +13,11 @@ flake8-tuple>=0.2.13
 bump2version
 hypothesis==3.44.14
 requests
-mypy==0.720
+mypy==0.670
```

Comment:
might also be nice to keep this up to date

Reply:
+1 ... we also use this mypy version in the client.

Reply:
Kept the newer `mypy`.